osquery/core/init.cpp

```diff
@@ -531,12 +554,14 @@ void Initializer::start() const {
   }
   initLogger(binary_);
 
+#ifndef WIN32
```

Comment:
Is this gate related to the process abstractions? Or just helpful since the distributed APIs are another can of porting worms?